boards/arm/stm32f3_disco/doc/index.rst

```diff
@@ -261,7 +261,7 @@ Again you have to use the adapted command for newer PCB revisions (E and newer):
 
 .. zephyr-app-commands::
    :zephyr-app: samples/hello_world
-   :board: stm32f3_disco@E
+   :board: stm32f3_disco_E
```

Comment:
Afaik this was correct, see https://docs.zephyrproject.org/latest/guides/porting/board_porting.html#multiple-board-revisions

Reply:
Thanks @gmarull, I was not aware of such specific revision mechanism.
@helo9 I am sorry for this confusion.

Reply:
No worries, I was a bit confused when I did not find the board revision anymore, but had no time to look deeper in that. Thanks.